static/js/settings_notifications.js

```diff
@@ -203,7 +209,29 @@ export function set_up(settings_panel) {
     });
 
     set_enable_marketing_emails_visibility();
+
     rerender_ui();
+
+    // Has to be after rerender_ui because relevant rows in table are added there
+    container.find(".unmute_stream").on("click", (e) => {
+        const row = $(e.currentTarget).parent().parent();
```

Comment:
We should directly use `closest` here instead of using parent two times. I think it will be `closest(".stream-row")`.